llpc/translator/lib/SPIRV/libSPIRV/SPIRVDecorate.h

```diff
@@ -61,6 +61,10 @@ class SPIRVDecorateGeneric : public SPIRVAnnotationGeneric {
   SPIRVDecorateGeneric(Op OC);
 
   SPIRVWord getLiteral(size_t) const;
+  const char *getLiteralString() const {
+    assert(Literals.size() > 0);
+    return reinterpret_cast<const char *>(&Literals[0]);
```

Comment:
nit: `.data()` gives you the pointer to the buffer: `return reinterpret_cast<const char *>(Literals.data());`